website/docs/d/batch_pool.html.markdown

```diff
@@ -43,6 +43,8 @@ The following attributes are exported:
 
 * `start_task` - A `start_task` block that describes the start task settings for the Batch pool.
 
+* `max_tasks_per_node` - The maximum number of tasks that can run concurrently on a single compute node in the pool.
```

Comment:
Can we update the documentation for the resource ?

Reply:
Missed adding the file when making the commit - sorry! Done now